app/views/pages/api_release_notes.html.haml

```diff
@@ -9,7 +9,67 @@
 
 %section.api-documentation
   %hr
-  %h2 Date TBC
+  %h2 17th September 2020
+  %ul.list-bullet
+    %li
+      New miscellaneous fee types
+      %br/
+      %br/
+      Three new fee types have been added for use on claims 
+      %br
+      %br
+      These will be available for claims with representation orders issued on or after the CLAR implementation date (17th September 2020).
+      %br
+      %br
+      The fee types will be available for AGFS final, supplementary and hardship claims (Paper Heavy and Unused Materials) and LGFS Final and Hardship claims (Unused Materials).
+      %br
+      %br
+      %table
+        %caption
+          New fee types
+        %thead
+          %tr
+            %th
+              Unique code
+            %th
+              Description
+            %th
+              Applicability
+            %th
+              Validations
+        %tbody
+          %tr
+            %td
+              MIPHC
+            %td
+              Paper heavy case
+            %td
+              AGFS Final, Supplementary and Hardship claims
+            %td
+              The new Paper Heavy fee type is not available for claims with offences in Bands 1, 6 and 9.
+            %th            
+          %tr
+            %td
+              MIUMU
+            %td
+              Unused material (upto 3 hours)
+            %td
+              AGFS Final, Supplementary and Hardship claims. LGFS Final and Hardship claims. Graduated Fee type cases only.
+            %td
+              Quantity of one only
+          %tr
+            %td
+              MIUMO
+            %td
+              Unused material (over 3 hours)
+            %td
+              AGFS Final, Supplementary and Hardship claims. LGFS Final and Hardship claims. Graduated Fee type cases only.
+            %td
+              Quantity of 3+ only
```

Comment:
have a look at the fee type validations ticket - there are more

Reply:
Have added validations from https://dsdmoj.atlassian.net/browse/CBO-1421